crates/proof-of-sql/src/proof_primitive/dory/dynamic_dory_compute_commitments_test.rs

```diff
@@ -0,0 +1,513 @@
+    assert_eq!(res[0].0, expected);
+
+    let expected: GT = Pairing::pairing(Gamma_1[1], Gamma_2[3]) * F::from(19)
+        + Pairing::pairing(Gamma_1[2], Gamma_2[3]) * F::from(20)
+        + Pairing::pairing(Gamma_1[3], Gamma_2[3]) * F::from(21)
+        + Pairing::pairing(Gamma_1[0], Gamma_2[4]) * F::from(22)
+        + Pairing::pairing(Gamma_1[1], Gamma_2[4]) * F::from(23)
+        + Pairing::pairing(Gamma_1[2], Gamma_2[4]) * F::from(24)
+        + Pairing::pairing(Gamma_1[3], Gamma_2[4]) * F::from(25)
+        + Pairing::pairing(Gamma_1[0], Gamma_2[5]) * F::from(26)
+        + Pairing::pairing(Gamma_1[1], Gamma_2[5]) * F::from(27)
+        + Pairing::pairing(Gamma_1[2], Gamma_2[5]) * F::from(28)
+        + Pairing::pairing(Gamma_1[3], Gamma_2[5]) * F::from(29)
+        + Pairing::pairing(Gamma_1[0], Gamma_2[6]) * F::from(30)
+        + Pairing::pairing(Gamma_1[1], Gamma_2[6]) * F::from(31)
+        + Pairing::pairing(Gamma_1[2], Gamma_2[6]) * F::from(32)
+        + Pairing::pairing(Gamma_1[3], Gamma_2[6]) * F::from(33)
+        + Pairing::pairing(Gamma_1[4], Gamma_2[6]) * F::from(34)
+        + Pairing::pairing(Gamma_1[5], Gamma_2[6]) * F::from(35)
+        + Pairing::pairing(Gamma_1[6], Gamma_2[6]) * F::from(36)
+        + Pairing::pairing(Gamma_1[7], Gamma_2[6]) * F::from(37);
+    assert_eq!(res[1].0, expected);
+
+    let expected: GT = Pairing::pairing(Gamma_1[1], Gamma_2[3]) * F::from(38)
+        + Pairing::pairing(Gamma_1[2], Gamma_2[3]) * F::from(39)
+        + Pairing::pairing(Gamma_1[3], Gamma_2[3]) * F::from(40)
+        + Pairing::pairing(Gamma_1[0], Gamma_2[4]) * F::from(41)
+        + Pairing::pairing(Gamma_1[1], Gamma_2[4]) * F::from(42)
+        + Pairing::pairing(Gamma_1[2], Gamma_2[4]) * F::from(43)
+        + Pairing::pairing(Gamma_1[3], Gamma_2[4]) * F::from(44)
+        + Pairing::pairing(Gamma_1[0], Gamma_2[5]) * F::from(45)
+        + Pairing::pairing(Gamma_1[1], Gamma_2[5]) * F::from(46)
+        + Pairing::pairing(Gamma_1[2], Gamma_2[5]) * F::from(47)
+        + Pairing::pairing(Gamma_1[3], Gamma_2[5]) * F::from(48)
+        + Pairing::pairing(Gamma_1[0], Gamma_2[6]) * F::from(49)
+        + Pairing::pairing(Gamma_1[1], Gamma_2[6]) * F::from(50)
+        + Pairing::pairing(Gamma_1[2], Gamma_2[6]) * F::from(51)
+        + Pairing::pairing(Gamma_1[3], Gamma_2[6]) * F::from(52)
+        + Pairing::pairing(Gamma_1[4], Gamma_2[6]) * F::from(53)
+        + Pairing::pairing(Gamma_1[5], Gamma_2[6]) * F::from(54)
+        + Pairing::pairing(Gamma_1[6], Gamma_2[6]) * F::from(55)
+        + Pairing::pairing(Gamma_1[7], Gamma_2[6]) * F::from(56);
+    assert_eq!(res[2].0, expected);
+}
+
+#[test]
+fn we_can_compute_an_empty_dynamic_dory_commitment() {
+    let public_parameters = PublicParameters::test_rand(5, &mut test_rng());
+    let setup = ProverSetup::from(&public_parameters);
+    let res = compute_dynamic_dory_commitments(&[CommittableColumn::BigInt(&[0, 0])], 0, &setup);
```

Comment:
Looks like the old test was wrong as well.
```suggestion
    let res = compute_dynamic_dory_commitments(&[CommittableColumn::BigInt(&[0; 0])], 0, &setup);
```